pyca/ingest.py

```diff
@@ -52,13 +52,13 @@ def ingest(event):
     # The ingest service to use is selected at random from the available
     # ingest services to ensure that not every capture agent uses the same
     # service at the same time
-    service = config('service-ingest')
-    service = service[randrange(0, len(service))]
-    logger.info('Selecting ingest service to use: ' + service)
+    service_url = service('ingest', True)
```

Comment:
Maybe be more explicit here, snce it can be missed very easily that this will always update the ingest server and therefore solves #166 and #168

```suggestion
    service_url = service('ingest', force_update=True)
```